idl/uuid.idl.hh

```diff
@@ -19,3 +19,7 @@ class UUID final {
 class table_id final {
     utils::UUID uuid();
 };
+
+class table_schema_version final {
+    utils::UUID uuid();
+};
```

Comment:
table_id and table_schema_version should be defined near their users, not in a generic uuid module. However, it's probably too hairy to untangle so we can live with it for now.

Reply:
yes, it is used in multiple places and even with the support for 'import' it is hard to decide who's the lowest level user for it so it can be defined there.

Reply:
The standard approach is to define their own modules. But let's not do it now.

Reply:
yes, I thought of `schema.idl.hh`, but we I agree it's an overkill for just those 2 basic types.